baseline.yaml

```diff
@@ -649,6 +649,30 @@ criteria:
     security_insights_value: # TODO
     scorecard_probe: # TODO
 
+- id: OSPS-DO-14
+    maturity_level: 3
+    category: Documentation
+    criteria: |
+       The project documentation MUST provide a 
+       descriptive statement when releases or 
+       versions are no longer supported and that 
+       will no longer receive security updates.  
+       
+       This should be provided both in human and 
+       machine-readable formats. 
```

Comment:
1. Do we mean "should" or "SHOULD" in the second sentence? If it's the "normal English 'should'", maybe we actually mean "MUST" instead? (Personally, I lean toward SHOULD, but I want to make sure I understand your intent)
2. As a rule, I don't like the idea of multi-sentence criteria and definitely not multi-paragraph (in the `<p>` sense, not in the English sense). Could it be be re-written as "The project documentation MUST provide human- and machine-readable descriptive statements when releases are no longer supported and will not longer receive security updates"? (As a note, I condensed "releases or versions" to be "releases" because I'm not entirely clear that there's a meaningful distinction between the two). Or, if the intent was for having both human- and machine-readable be a strong suggestion but not a requirement, then we should move the second sentence into the implementation (which it sort of already is)